src/sites_toolbox.py

```diff
@@ -81,11 +81,23 @@ def get_site_group_details(session, site_group_name):
     )
     site_group_sites = [
         {"site_uuid": str(site.site_uuid), "client_site_id": str(site.client_site_id)}
-        for site in site_group_uuid.sites]
+        for site in site_group_uuid.sites
+    ]
     site_group_users = [user.email for user in site_group_uuid.users]
     return site_group_sites, site_group_users
 
 
+# change site group for user
+def change_user_site_group(session, email, site_group_name):
+    update_user_site_group(
```

Comment:
add function description and typing

Reply:
"""
Change user to a specific site group name

:param session: the database session
:param email: the email of the user 
...